pandas/core/indexes/accessors.py

```diff
@@ -126,6 +126,26 @@ class DatetimeProperties(Properties):
     """
 
     def to_pydatetime(self):
+        """
+        Return DatetimeIndex as an object ndarray.
+
+        Returns
+        -------
+        ndarray of object dtype
+
+        See Also
+        --------
+        pandas.DatetimeIndex.to_pydatetime : Convert a Timestamp object to a native
+            Python datetime object.
+
+        Examples
+        --------
+        >>> df = pd.Series(pd.date_range('20180310', periods=2))
+        >>> type(df)
+        <class 'pandas.core.series.Series'>
+        >>> type(df.dt.to_pydatetime())
```

Comment:
I would actually show the output here, instead of inspecting the type (that will also show it is an array, but also show it holds datetime.datetime objects)